battleship.py

```diff
@@ -135,13 +137,13 @@ def addShips(grid, numShips):
 Returns: None
 '''
 def drawGrid(data, canvas, grid, showShips):
-    for i in range(data["no.of rows"]):
-        for j in range(data["no.of cols"]):
+    for row in range(data["no.of rows"]):

```

Comment:
make the mentioned changes for variables

Reply:
changed mam